stats/templates/stats/index.html

```diff
@@ -25,7 +25,14 @@ <h1>Stats</h1>
         <th scope="col" style="width: 250px"><a
             href="?order_by={% if request.GET.order_by == 'completed_challenges' %}-{% endif %}completed_challenges">Challenges completed
           {% if 'username' not in request.GET.order_by or not request.GET.order_by %}
-            <div class="arrow-{% if request.GET.order_by == 'completed_challenges' %}up{% else %}down{% endif %} ml-2 mb-1"></div>
+            <!--<div class="arrow-{% if request.GET.order_by == 'completed_challenges' %}up{% else %}down{% endif %} ml-2 mb-1"></div>-->
```

Comment:
It is a good idea to show the sorting arrow next to the column name, if the list is being sorted based on that field. Currently, when sorting by "Completed challenges", the implementation does not show an arrow next to it, and a downward arrow is shown next to "Score".